charts/timescaledb-multinode/README.md

```diff
@@ -38,6 +38,25 @@ Alternatively, a YAML file that specifies the values for the parameters can be p
 ```console
 helm upgrade --install my-release -f myvalues.yaml .
 ```
+### Secret override
+
+In order to not have secrets stored in git, you can manually generate secrets for timescaledb-access endpoint and timescaledb-data endpoint.
+
+In values.yaml set:
+```console
+credentials:
+  fromValues: false
+```
+
+Then generate the secrets
+```console
+random_password () { < /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c32; }
```

Comment:
this does not belong here

Reply:
Can you give more details ?

Reply:
this function is already defined above, where it is being used
you are not using it here, so why defining it again ?